apps/owmweather/interface.html

```diff
@@ -2,35 +2,34 @@
   <head>
     <link rel="stylesheet" href="../../css/spectre.min.css">
   </head>
-  <body>   
+  <body>
     <h3>Set OpenWeatherMap (OWM) API key</h3>
     <p><input id="apikey" onkeyup="checkInput()"  style="width:90%; margin: 3px"></input><button id="upload" class="btn btn-primary">Save key</button></p>
-    
+
     <h4>Where to get your personal API key?</h4>
     <p>Go to <a href="https://home.openweathermap.org/users/sign_up">https://home.openweathermap.org/users/sign_up</a> and sign up for a free account.<br>
     After registration you can login and optain your personal API key.</p>
-    
+
 
     <script src="../../core/lib/interface.js"></script>
 
     <script>
-    
+
        function checkInput() {
-        if(document.getElementById("apikey").value==="") { 
-                document.getElementById('upload').disabled = true; 
-            } else { 
+        if(document.getElementById("apikey").value==="") {
+                document.getElementById('upload').disabled = true;
+            } else {
                 document.getElementById('upload').disabled = false;
             }
         }
         checkInput();
-    
+
         var settings = {};
         function onInit(){
           console.log("Loading settings from BangleJs...");
           try {
-            Util.readStorage("owmweather.json", data=>{
-              if(data.length > 0){
-                settings = JSON.parse(data);
+            Util.readStorageJSON("owmweather.json", settings=>{
+              if("settings){
```

Comment:
Minor typo
```suggestion
              if(settings){
```

Reply:
Ouch - thanks! I wonder if we can lint the JS in HTML files somehow?

Reply:
Good idea - I suppose we could add a check that does something like this? (except a little more refined)

```sh
awk '/<script>/, /<\/script>/' < html | sed '1d; $d' | node -c
```